packages/core/src/radio-group/radio.scss

```diff
@@ -1,58 +1,132 @@
+$variants: default, valid, invalid;
+$variant-selectors: ':not([valid],[invalid])', ':where([valid])', ':where([invalid])';
+$states: unchecked, checked;
+$state-selectors: '', ':where(:--checked, :--indeterminate, [state--checked], [state--indeterminate])';
+$sizes: md, lg;
+
 :host {
-  --_background: #fff;
-  --_foreground: #fff;
-  --_border: 1px solid var(--_border-color, #ccc);
-  --_gap: 0.25rem;
-  --_line-height: 1.5rem;
-  --_padding: 2px;
-  --_size: 1rem;
+  --_border-radius: var(--sl-border-radius-circle);
+  --_border-size: var(--sl-border-width-input-border);
+  --_border: var(--_border-size) solid var(--_border-color);
+  --_gap: var(--sl-space-input-gap-md);
+  --_line-height: var(--sl-text-typeset-line-height-md);
+  --_padding: var(--sl-space-input-block-md) var(--sl-space-input-inline-md) var(--sl-space-input-block-md) 0;
+  --_path-length: 22.910259;
+  --_stroke-dashoffset: var(--_path-length);
+  --_transition: 200ms ease-in-out;
+  --_icon-size: 16px;
 }
 
 :host {
   align-items: start;
-  cursor: pointer;
-  display: inline-flex;
+  display: flex;
+  flex-direction: column;
   gap: var(--_gap);
   line-height: var(--_line-height);
   outline: none;
+  padding: var(--_padding);
```

Comment:
So recently I've been using `padding` less and more `padding-inline` & `padding-block` (same for `margin`). By using the logical versions of those properties, it's easier also to customise them since you can have separate variables for the inline & block versions. That was always a (potential) problem with a single css variable.

Reply:
Not saying you need to change anything in this case :)